### PR TITLE
Fix social icon becomes white issue

### PR DESCRIPTION
**Pull request by image7**
Fix: #78

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -210,6 +210,7 @@ div#content_block { width: 70% }
 
 /*Facebook Button Frontend Styles*/
 .rtsocial-fb-button{ background-position: 0 -172px; display: block; height: 24px; width: 50px; text-indent: -9999px; text-transform: uppercase; margin: 0 auto;}
+.rtsocial-fb-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -172px no-repeat !important;}
```
Comment (robicse11127) [inline styles/style.css:213]: Use of `!important` is causing the PHPCS test failure.

File: styles/style.css [hunk 2]
```diff
@@ -230,6 +231,7 @@ div#content_block { width: 70% }
 .rtsocial-twitter-vertical-button { text-align: center; }
 .rtsocial-twitter-button,
 #rtsocial-ord-tw label { background-position: 0 -557px;  display: block; float: left; height: 21px; margin: 0; padding: 0; width: 58px; text-indent: -9999px; }
+.rtsocial-twitter-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -557px no-repeat !important;}
```
Comment (robicse11127) [inline styles/style.css:234]: Use of `!important` is causing the PHPCS test failure.

File: styles/style.css [hunk 3]
```diff
@@ -239,13 +241,15 @@ div#content_block { width: 70% }
 /* Pinterest Button */
 .rtsocial-pinterest-button,
 #rtsocial-ord-pin label { background-position: 0 -598px; border: 1px solid #c9c5c5; border-color: #E8E4E4 #C9C5C5 #C9C5C5; border-radius: 4px 4px 4px 4px; display: block; height: 22px; text-indent: -9999px; text-transform: uppercase; width: 47px; margin: 0 auto; }
+.rtsocial-pinterest-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -598px no-repeat !important;}
```
Comment (robicse11127) [inline styles/style.css:244]: Use of `!important` is causing the PHPCS test failure.

File: styles/style.css [hunk 4]
```diff
@@ -239,13 +241,15 @@ div#content_block { width: 70% }
...
 
 .rtsocial-pinterest-button:hover,
 #rtsocial-ord-pin label:hover{ background-position: 0 -640px; border: 1px solid #B8B4B4; }
 
 /* Linked In Button */
 .rtsocial-linkedin-button,
 #rtsocial-ord-lin label{ background-position: 0 -682px; border-color: #E2E2E2 #BFBFBF #B9B9B9; border-style: solid; border-width: 1px; border-radius: 2px; display: block; height: 18px; margin: 0 auto; text-indent: -9999px; text-transform: uppercase; width: 59px; }
+.rtsocial-linkedin-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -682px no-repeat !important;}
```
Comment (robicse11127) [inline styles/style.css:252]: Use of `!important` is causing the PHPCS test failure.

**Review by robicse11127 — APPROVED** [review]

File: styles/style.css (hunk 2 above)
Comment (image7) [reply styles/style.css:234]: The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

File: styles/style.css (hunk 3 above)
Comment (image7) [reply styles/style.css:244]: The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

File: styles/style.css (hunk 4 above)
Comment (image7) [reply styles/style.css:252]: The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

Comment (image7) [reply styles/style.css:252]: The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

File: styles/style.css (hunk 1 above)
Comment (image7) [reply styles/style.css:213]: The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

**Pull request merged**